src/components/Editor/Footer.js

```diff
@@ -54,14 +46,23 @@ const SourceFooter = React.createClass({
       return;
     }
 
-    return debugBtn(
-      this.onClickPrettyPrint,
-      "prettyPrint",
-      classnames({
-        active: sourceLoaded,
-        pretty: isPretty(selectedSource.toJS())
-      }),
-      L10N.getStr("sourceFooter.debugBtnTooltip")
+    const tooltip = L10N.getStr("sourceFooter.debugBtnTooltip");
+    const type = "prettyPrint";
+    const className = classnames({
+      action: true,
+      type,
+      active: sourceLoaded,
+      pretty: isPretty(selectedSource.toJS())
+    });
+
+    return dom.button({
+        onClick: this.onClickPrettyPrint,
+        className,
```

Comment:
I'm infavor of inlining this, even though i suppose it'll make the JSX harder in the future...